doc/cli.markdown

```diff
@@ -636,6 +638,10 @@ environment variable numeric database ID
 
 Selects a device environment variable instead of an application environment variable
 
+#### -c, --config
+
+Selects a configuration variable instead of an application/device environment variable
```

Comment:
Suggestion:
* Here it could say simply `Selects a configuration variable instead of an environment variable`, which I think avoids implying that `application/device` is a distinction that applies to `environment` variables but not to `configuration` variables. (Alternatively but more verbose: `Selects an application/device configuration variable instead of an application/device environment variable`)
* A few paragraphs above, where the overall command description currently reads:
```
Remove an environment variable from an application or device, as selected
by command-line options.
...
The --device option selects a device instead of an application.
```
Perhaps it should instead read:
```
Remove a configuration or environment variable from an application or device,
as selected by command-line options.
...
The --device option selects a device instead of an application.
The --config option selects a config var instead of an env var.
```